extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -865,6 +865,17 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       // creative will be centered instead of left-aligned.
       this.element.removeAttribute('height');
       setStyles(this.element, {width: `${size.width}px`});
+    } else if (this.isFluidRequest_ && this.isVerifiedAmpCreative) {
+      // This is an AMP fluid creative that will be rendered in a friendly
```

Comment:
Do we want/need to run this as an experiment?

Reply:
I assumed it wasn't necessary since there aren't any fluid AMP creatives and the change is benign enough. Glad to guard it behind an experiment if you think otherwise, though.